app/services/editions/publishing/PublishedIssueFormatters.scala

```diff
@@ -4,10 +4,10 @@ import model.editions._
 import play.api.libs.json.Json
 
 object PublishedIssueFormatters {
-  implicit val publishedImageFormat = Json.format[PublishedImage]
-  implicit val publishedFurnitureFormat = Json.format[PublishedFurniture]
-  implicit val publishedArticleFormat = Json.format[PublishedArticle]
-  implicit val publishedCollectionsFormat = Json.format[PublishedCollection]
-  implicit val publishedFrontsFormat = Json.format[PublishedFront]
-  implicit val publishedIssueFormat = Json.format[PublishedIssue]
+  implicit val publishedImageWrites = Json.writes[PublishedImage]
```

Comment:
why?

Reply:
We don't use them for reading in. We can make it explicit. If we want to be able to use them for reading in we need to add more tests to ensure that the format doesn't change.